app/(dashboard)/layout.tsx

```diff
@@ -5,18 +5,62 @@
 //
 // SPDX-License-Identifier: MIT
 //
+import { Home } from 'lucide-react'
+import Link from 'next/link'
 import { type ReactNode } from 'react'
-import { authenticatedOnly } from '../../modules/firebase/guards'
+import { LogoType } from '@/components/icons/LogoType'
+import { getAuthenticatedOnlyApp } from '@/modules/firebase/guards'
+import { getUserInfo } from '@stanfordbdhg/design-system/modules/auth/user'
+import {
+  DashboardLayout as DashboardLayoutBase,
+  MenuItem,
+  PageTitle,
+} from '@stanfordbdhg/design-system/molecules/DashboardLayout'
+import { User } from './User'
 
 interface DashboardLayoutProps {
   children?: ReactNode
 }
 
 export const dynamic = 'force-dynamic'
 
+const MenuLinks = () => (
+  <>
+    <MenuItem href="/" label="Home" icon={<Home />} isActive />
+  </>
+)
+
 const DashboardLayout = async ({ children }: DashboardLayoutProps) => {
-  await authenticatedOnly()
-  return children
+  const { currentUser } = await getAuthenticatedOnlyApp()
+
+  const user = <User user={getUserInfo(currentUser)} />
+
+  return (
+    <DashboardLayoutBase
```

Comment:
> 1. Would be good to move this closer to the top, maybe below the horizontal title line with some small padding?

Sure, that works too. 

[Decrease vertical space](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/pull/13/commits/b7c9c4254ba8a4bff9076793cbf7ade694b37ea3)

> 2. I have the feeling this main bar has a bit too much vertical (and maybe slightly horizontal) spacing, might be cool to explore how this can be reduced a bit, e.g. taking the GitHub Main Bar as some inspiration.

Header bar is just mostly empty for now. It can contain way more, like action button, search input, controls, tabs, breadcrumbs. I think your feeling might be caused because of it's current emptiness. When you compare it against GitHub Main bar, it's way more crowded, but at the same GH's bar is even more vertical (86px ours vs 106px GH). I'd keep as is for now and possibly iterate later if needed.

> 3. I would suggest making the sidebar a bit wider in its expanded form. It might be a bit of a bias, but it would be great if my name would fit without truncation 😄

Mine doesn't fit either 😄 It's not easy to fit our names! I would love not to expand sidebar more, because it's going to occupy more working space. 

I played around making dashboard more compact in general. Let me know what you think
[Make Dashboard more compact](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/pull/13/commits/bb71ceaee180609e2b3515b763308a4b35cbe73a)

I was very generous with white space, but more compact design is more universal.

> I have also encountered a small bug

Fixed, thanks for catching it. It was actually caused by design system vs app classes clash. I might resolve it different way in the future, but for now this is ok. [Fix classes clash](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/pull/13/commits/116fe7e4414934b21635149c859a8d96678910ab)

Reply:
Thank you for looking at all of this! Will check this out later today but feel free to merge the PR if you feel comfortable in merging it before 👍